FormalConjectures/ErdosProblems/11.lean

```diff
@@ -0,0 +1,50 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+import Mathlib.Data.Nat.Squarefree
+
+/-!
+# Erdős Problem 11
+
+*Reference:* [erdosproblems.com/11](https://www.erdosproblems.com/11)
+-/
+/--
+Is every odd n the sum of a squarefree number and a power of 2?
+-/
+
+@[category research open, AMS 11]
+theorem erdos_11 (n : ℕ) (hn : Odd n) :
+    ∃ k l : ℕ , Squarefree k ∧ n = k + 2^l := by
+  sorry
+
+/--
+Erdős often asked this under the weaker assumption that n
+is not divisible by 4
+.
+-/
+@[category research open, AMS 11]
+theorem erdos_11_i (n : ℕ) (hn : Odd n) (hn_2 : n % 4 ≠ 0):
+    ∃ k l : ℕ , Squarefree k ∧ n = k + 2^l := by
+  sorry
+
+/--
+Erdős thought that proving this with two powers of 2 is perhaps easy, and could prove that it is true (with a single power of two) for almost all n.
```

Comment:
```suggestion
Erdős thought that proving this with two powers of 2 is perhaps easy, and could prove that it is true (with a single power of two) for almost all n.
```

Could "and could prove that it is true (with a single power of two) for almost all n." be made into a solved variant?
(Or add a TODO)

Reply:
In [Er90] (using the website referencing), he writes

> 11. Is it true that if n = 0 (mod 4) then there is a squarefree natural
number theta such that n = 2^k + theta ? I could only prove that almost all
integers n = 0 (mod 4) can be written in the form 2^k + theta.

I cannot find where he proved this though.